src/web/app/pages-admin/admin-notifications-page/admin-notifications-page.component.spec.ts

```diff
@@ -20,11 +55,120 @@ describe('AdminNotificationsPageComponent', () => {
 
   beforeEach(() => {
     fixture = TestBed.createComponent(AdminNotificationsPageComponent);
+    notificationService = TestBed.inject(NotificationService);
+    statusMessageService = TestBed.inject(StatusMessageService);
     component = fixture.componentInstance;
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default view', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap when notifications are loading', () => {
+    component.isNotificationLoading = true;
+    fixture.detectChanges();
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap when notifications failed to load', () => {
+    component.hasNotificationLoadingFailed = true;
+    fixture.detectChanges();
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap when notification edit form expanded', () => {
+    component.isNotificationEditFormExpanded = true;
```

Comment:
Currently, the snapshot for this is exactly the same a that for the case where notifications are loading; in order for the notification form to be displayed, `isNotificationLoading` has to be set to `false` here.

Reply:
Thanks for the catch!